packages/react-app/src/components/parts/ColorModeButton.tsx

```diff
@@ -1,15 +1,20 @@
 import { Button, useColorModeValue } from '@chakra-ui/react'
+import NextLink from 'next/link'
 
 const ColorModeButton = ({
   children,
+  url
 }: {
-  children?: React.ReactNode
+  children?: React.ReactNode,
+  url: string
```

Comment:
This breaks existing `ColorModeButton` functionality which is used on other parts of the site. Please see my other comment about possible using the `AccessibleLink` in the `Filters` component instead of adding link functionality to `ColorModeButton`